CD2/enrollments.md

```diff
@@ -0,0 +1,24 @@
+#Enrollments Queries
```

Comment:
improvements on Markdown syntax needed. Here is the [Markdown cheatsheet](https://www.markdownguide.org/cheat-sheet/)

insert a space in-between the markdown symbols and the text. For example, use "# Enrollments" vs "#Enrollments".

You can do preview using VisualStudio or https://markdownlivepreview.com/